File: DSA_Questions/common_dsa_questions/common_dsa_questions/CountSuchPairs.java
package common_dsa_questions;

public class CountSuchPairs {

	public static void main(String[] args) {
		
		
		int[] arr = {3,0,6,2,7};
		
		int k = 9;
		
		int n = arr.length;
		
		int count=0;
		
		for(int i=0; i<n; i++)
		{
			for(int j=i; j<n; j++)
			{
				if(arr[i]+arr[j] == k)
				{
					count++;
				}
			}
		}
		
		System.out.println(count);

	}

}

File: DSA_Questions/bin/Python/CountSuchPairs.py
n = 5
k = 9
array = [3, 0, 6, 2, 7]


def countSuchPairs(N, K, arr):
    count = 0
    for i in range(0, N):
        for j in range(i + 1, N):
            if arr[i] + arr[j] == K:
                count += 1
    print(count)


countSuchPairs(n, k, array)
